[2024-01-23T18:44:05.007+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_stackoverflow_data.get_stackoverflow_data scheduled__2022-11-27T00:00:00+00:00 [queued]>
[2024-01-23T18:44:05.065+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_stackoverflow_data.get_stackoverflow_data scheduled__2022-11-27T00:00:00+00:00 [queued]>
[2024-01-23T18:44:05.066+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 6
[2024-01-23T18:44:05.297+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): get_stackoverflow_data> on 2022-11-27 00:00:00+00:00
[2024-01-23T18:44:05.391+0000] {standard_task_runner.py:60} INFO - Started process 4695 to run task
[2024-01-23T18:44:05.484+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'fetch_stackoverflow_data', 'get_stackoverflow_data', 'scheduled__2022-11-27T00:00:00+00:00', '--job-id', '672', '--raw', '--subdir', 'DAGS_FOLDER/loading_data_to_aws_S3.py', '--cfg-path', '/tmp/tmp43c3ztyy']
[2024-01-23T18:44:05.524+0000] {standard_task_runner.py:88} INFO - Job 672: Subtask get_stackoverflow_data
[2024-01-23T18:44:05.907+0000] {task_command.py:423} INFO - Running <TaskInstance: fetch_stackoverflow_data.get_stackoverflow_data scheduled__2022-11-27T00:00:00+00:00 [running]> on host 9924936b75b5
[2024-01-23T18:44:07.518+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='charlotte' AIRFLOW_CTX_DAG_ID='fetch_stackoverflow_data' AIRFLOW_CTX_TASK_ID='get_stackoverflow_data' AIRFLOW_CTX_EXECUTION_DATE='2022-11-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-11-27T00:00:00+00:00'
[2024-01-23T18:44:10.592+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-01-23T18:44:10.700+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=fetch_stackoverflow_data, task_id=get_stackoverflow_data, execution_date=20221127T000000, start_date=20240123T184405, end_date=20240123T184410
[2024-01-23T18:44:10.936+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-23T18:44:11.253+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
